Tao_Current.txt

```diff
@@ -1,73 +1,84 @@
 Niels ten Oever and Kathleen Moriarty, Editors
+
 About This Document
 
-The current version of this web page can always be found at http://www.ietf.org/tao.html; that page can also be retrieved protected by TLS. To contribute to this document or to discuss its content, please join the "tao-discuss" mailing list. A history of the major versions of the Tao can be found here. This particular version was created on 2012-11-02.
+This is the upcoming revision of the Tao which can be found at [http://www.ietf.org/tao-possible-revision.html]
+
+The current version of this web page can always be found at [https://www.ietf.org/tao.html]. To contribute to this document or to discuss its content, please join the "tao-discuss" mailing list. A history of the major versions of the Tao can be found here. This particular version was created on 2018-06-26.
 
 This web page is in English. There is a list of translations of the Tao of the IETF.
+
 1. Introduction
 
-Since its early years, attendance at Internet Engineering Task Force (IETF) face-to-face meetings has grown phenomenally. Many of the attendees are new to the IETF at each meeting, and many of those go on to become regular attendees. When the meetings were smaller, it was relatively easy for a newcomer to get into the swing of things. Today, however, a newcomer meets many more new people, some previously known only as the authors of documents or thought-provoking email messages.
+Since its early years, attendance at Internet Engineering Task Force (IETF) face-to-face meetings has grown phenomenally and is now stabilizing. Many of the attendees are new to the IETF at each meeting, and many of those go on to become regular attendees. When the meetings were smaller, it was relatively easy for a newcomer to adjust. Today, however, a newcomer meets many more new people, some previously known only as the authors of documents or thought-provoking email messages.
```

Comment:
https://datatracker.ietf.org/stats/meeting/overview/

Maybe it can be:
Proposed text:
Since its early years, attendance at Internet Engineering Task Force (IETF) face-to-face meetings has grown phenomenally and is now averaging between 1100 and 1400 participants [https://datatracker.ietf.org/stats/meeting/overview/].

Reply:
I still think using specific numbers is unwarranted. The Tao doesn't get updated very often, so better to make this generic enough that it won't be out of date in a year if participation trends change.

Reply:
I hope it is okay that we keep it in here, since we agreed we're going to update the Tao more regularly anyhow.